react/features/base/participants/functions.js

```diff
@@ -94,6 +95,21 @@ export function getLocalParticipant(stateful: Object | Function) {
     return participants.find(p => p.local);
 }
 
+/**
+ * Normalizes a display name so then no invalid values (padding, length...etc)
+ * can be set.
+ *
+ * @param {string} name - The display name to set.
+ * @returns {string}
+ */
+export function getNormalizedDisplayName(name: string) {
+    if (!name || !name.trim()) {
+        return undefined;
```

Comment:
Should an empty string not be returned?

Reply:
As far as I know, not, because we want to delete its value. I've seen occasions where an empty string just set it an empty string and not something unset. What do you think?

Reply:
I'm wondering what the difference is between empty and and undefined, because likely in the UI we would default to an empty string anyway. Empty string seems safer to me, as an empty string and undefined go through the same common truthy check leading to false. I guess what's most important is that there is consistency now in that it's not sometimes empty string but rather always undefined.